src/main/kotlin/com/zopa/ktor/opentracing/Route.kt

```diff
@@ -0,0 +1,36 @@
+package com.zopa.ktor.opentracing
+
+import io.ktor.http.HttpMethod
+import io.ktor.routing.RoutingPath
+import io.ktor.routing.RoutingPathSegment
+import io.ktor.routing.RoutingPathSegmentKind
+
+class Route(
+        val method: HttpMethod,
+        val path: String
+) {
+    val segments: List<RoutingPathSegment> = RoutingPath.parse(path).parts
+
+    fun matchToCallRoute(call: Route): RouteMatchResult {
```

Comment:
You should be able to only return a List<Tag>, because when `result` is false it will always be empty.

Reply:
Ah yes you're right, because only routes with params matter!